src/main/java/com/auth0/StorageUtils.java

```diff
@@ -0,0 +1,26 @@
+package com.auth0;
+
+import org.apache.commons.codec.binary.Base64;
+
+import java.security.SecureRandom;
+
+class StorageUtils {
+
+    private StorageUtils() {}
```

Comment:
could be an abstract class and this line can go away

Reply:
Similar to my comment above, I'd prefer to not have an abstract class unless its purpose is to be extended.